pkgs/applications/editors/lite-xl/default.nix

```diff
@@ -0,0 +1,132 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, SDL2
+, lua52Packages
+, pkg-config
+, makeWrapper
+, openlibm
+, meson
+, pcre2
+, freetype
+, git
+, reproc
+, agg
+, ninja
+, curl
+, unzip
+, writeScriptBin
+}:
+let
+  build = writeScriptBin "build" ''
```

Comment:
Probably can put the script in `buildPhase` so we don't need to add something to `buildInputs` or use `writeScriptBin`.

Reply:
didn't knew that I could do full bash scripting on the buildPhase. fixing it now!